app/docs/how-to/streamline-curbside.md

```diff
@@ -0,0 +1,255 @@
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/curbside-form-name-description.png" width="30%">
+</p>
+
+#### Step 4: Define touchpoint schema
+
+Next, we are going to define a touchpoint [schema](/reference/schemas/). Schema is an exteremely important part of a touchpoint that defines what kind of data is collected from people interacting with your touchpoint.
+
+To get started, select **Define your own schema** from **Schema** dropdown list and name your new schema.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-define-schema.png" width="75%">
+</p>
+
+The Routegy schema is defined by a [JSON Schema](https://json-schema.org/) document in YAML format. To collect customer's name, car information and order type, we are going to use the following schema:
+
+```yaml
+type: object
+attrs:
+  successMessage: Thank You!
+order:
+  - order_type
+  - name
+  - car
+required:
+  - name
+  - car
+  - order_type
+properties:
+  car:
+    type: string
+    attrs:
+      placeholder: E.g. White Honda Accord
+    title: Please give us some details about your car
+  name:
+    type: string
+    attrs:
+      placeholder: E.g. John
+    title: What is the name on the order?
+  order_type:
+    enum:
+      - By Phone
+      - Online
+      - GrubHub
+      - Other
+    type: string
+    attrs:
+      type: radio
+    title: How did you place your order?
+```
+
+This YAML document defines a simple schema with **car**, **name** and **order_type** fields, and it can be easily tailored to meet the needs of your business. For instance, you can easily modify titles of these fields by changing **title** attribute under each individual field.
+
+To apply this document to our newely created schema, scroll down to **Definition (YAML)** section of the form, and copy/paste the YAML code above replacing the original content.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-schema-yaml.png" width="75%">
+</p>
+
+Once done, you can click the **Preview** tab to inspect the contents of the schema form.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-schema-preview.png" width="75%">
+</p>
+
+#### Step 5: Save and test your touchpoint
+
+If you are happy with the name, description and schema, click **Create touchpoint** button to save the changes, and you will be presented with a QR code and a URL for your new touchpoint.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-qr.png" width="75%">
+</p>
+
+Scan the code with a camera app on your phone (or click the link) to make sure that the your new curbside pickup touchpoint looks and works just like you defined. You can always go back and tweak the name, description and schema if needed.
```

Comment:
Link the "camera app" phone text to the supported devices doc.

Reply:
Condensed: 
> Click the link or scan the code with the camera app on your phone to check it out! Want to make some changes? Don't worry, you can always go back and tweak it at anytime.

Reply:
Took your text.